controller/rest/policy.go

```diff
@@ -853,7 +862,16 @@ func insertPolicyRule(scope string, w http.ResponseWriter, r *http.Request, inse
 	crhs = append(crhs[:toIdx], append(news, crhs[toIdx:]...)...)
 
 	// Put policy rule heads
-	clusHelper.PutPolicyRuleListTxn(txn, crhs)
+	if err := clusHelper.PutPolicyRuleListTxn(txn, crhs); err != nil {
+		log.WithFields(log.Fields{"error": err}).Error("PutPolicyRuleListTxn")
+		lastError = err
+	}
+
+	if lastError != nil {
+		log.WithFields(log.Fields{"error": lastError}).Error("")
```

Comment:
FYI: code like
`log.WithFields(log.Fields{"error": err}).Error("")`
can just be changed to
`log.WithFields(log.Fields{"error": err}).Error()`
(the empty string parameter for log.Error()/Warn()/Info()/Debug() is unnecessary)